libs/openFrameworks/utils/ofConstants.h

```diff
@@ -188,7 +188,7 @@ enum ofTargetPlatform{
 	#endif
 
 	#if defined(__OBJC__) && !__has_feature(objc_arc)
-		#error "Please enable ARC (Automatic Reference Counting) at the project level"
+// 		#error "Please enable ARC (Automatic Reference Counting) at the project level"
 	#endif
```

Comment:
@dimitre want to change this to #warning and then we can merge?

Reply:
Yes, I don't now how to do it. any suggestion? 
#warning does nothing on xcode
so maybe it is #pragma warning?